tls/s2n_ocsp_stapling.c

```diff
@@ -27,24 +27,14 @@
 
 int s2n_server_status_send(struct s2n_connection *conn)
 {
-    GUARD(s2n_server_certificate_status_send(conn, &conn->handshake.io));
+    if (s2n_server_can_send_ocsp(conn)) {
+        GUARD(s2n_server_certificate_status_send(conn, &conn->handshake.io));
+    }
 
     return 0;
```

Comment:
return S2N_SUCCESS? :)

Reply:
I can do that, but I generally haven't been updating the legacy methods as I go unless absolutely necessary. They're going to be deleted when I finish this migration.

Reply:
Ah no worries then.